### PR TITLE
SAK-32578 updated OptionsBean.java options.jsp sections.properties

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

**Comment by ncaidin** [comment]
The change makes sense to me. But maybe worth getting a little more feedback on the wording ? Personally I like the "Manually Manage" button rather than a plain vanilla Ok or Submit. Makes one realize there is an impact when clicking.

File: sections/sections-app-util/src/bundle/sections.properties [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #Tue Jan 25 16:42:46 CET 2011
+options_confirmInternal=Manually manage avoids groups updates for registration system. Do you want to continue?
```
Comment (ottenhoff) [inline sections/sections-app-util/src/bundle/sections.properties]: Manually manage will prevent group membership updates from an automated enrollment system. Do you want to continue?

**Comment by mpellicer** [comment]
@joaquinmarques Seems this PR is wrong as contains more than your development, please close this PR, keep your changes, delete your branch and create a new branch/PR with the changes of SAK-32578.

Feel free to request help!

Thanks

**Comment by joaquinmarques** [comment]
This PR is wrong as contains more than my development, it is necessary close this PR

**Pull request closed without merging**